README.md

```diff
@@ -62,6 +62,10 @@ Once toolchain and other dependencies are in place, run `make build` to build, a
 `stratisd` executable in `./target/x86_64-unknown-linux-gnu/debug` as root.
 Pass the `--help` option for more information on additional developer options.
 
+##### Building tests
+The Makefile provides a target, build-tests which allows compiling the
+tests without running any of them, as a convenience to developers.
```

Comment:
Also, I don't see much interest in building tests without running them. Any point?

Reply:
Some people don't like to run the tests directly on their machines, just in case. But, on the other hand, they'ld like to know the tests all compile before pushing and getting the CI started.

Reply:
Then I suggest "as a convenience to lazy developers". :rofl: